Download Expression Matrix for Scanpy/Download SmartSeq2 Expression Matrix as Input to Scanpy.ipynb

```diff
@@ -16,7 +16,7 @@
   },
```

Comment:
<p>i think this text block needs to be updated to reflect the code below. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCellAtlas/data-consumer-vignettes/pull/67/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
👍